wifiphisher/common/phishingpage.py

```diff
@@ -31,68 +26,44 @@ def config_section_map(config_file, section):
 
 
 class InvalidTemplate(Exception):
-    """ Exception class to raise in case of a invalid template """
+    """Exception class to raise in case of a invalid template."""
 
-    def __init__(self):
-        Exception.__init__(self, "The given template is either invalid or " +
-                           "not available locally!")
+    pass
 
 
 class PhishingTemplate(object):
-    """ This class represents phishing templates """
+    """This class represents phishing templates."""
 
     def __init__(self, name):
-        """
-        Construct object.
-
-        :param self: A PhishingTemplate object
-        :type self: PhishingTemplate
-        :return: None
-        :rtype: None
-        .. todo:: Maybe add a category field
-        """
-
-        # setup all the variables
-
+        # type: (str) -> None
+        """Intialize the class with all arguments."""
         config_path = os.path.join(constants.PHISHING_PAGES_DIR, name,
                                    'config.ini')
```

Comment:
You may add the following snippet from the docs here:

  "A config.ini file lies in template’s root directory and its contents can be divided into two sections:

  <b>info</b>: This section defines the scenario’s characteristics.
  * Name (mandatory): The name of the phishing scenario
  * Description (mandatory): A quick description (<50 words) of the scenario
  * PayloadPath (optional): If the phishing scenario pushes malwares to victims, users can insert the absolute path of the malicious executable here


<b>context</b>: This section is optional and holds user-defined variables that may be later injected to the template.
  Here’s an example of a config.ini file:

  \> \# This is a comment 
  \> [info]
  \> Name: ISP warning page
  \> Description: A warning page from victim's ISP asking for DSL credentials
  \>
  \> [context]
  \> victim_name: John Phisher
  \> victim_ISP: Interwebz
"